HttpListener.cs

```diff
@@ -8,7 +8,7 @@ static class HttpListener
         public static void Init()
         {
             Console.WriteLine("Setting up listener");
-            listener.Prefixes.Add("http://localhost:5000/api/scores");
+            listener.Prefixes.Add("http://172.24.3.145:5000/api/scores");
```

Comment:
Good job on this! But maybe you can extract local IP as a string, it would be easier to change for us ;)